js/mainSpeakers.js

```diff
@@ -0,0 +1,87 @@
+const speakers = [
+  {
+    speakerImage: './images/speaker_01.png',
+    speakerName: 'David Fitch',
+    profession: 'Storyteller Panel:',
+    discription: 'Karen Wilk , David Lundberg, Taehoo Lee6',
+  },
+  {
+    speakerImage: './images/speaker_02.png',
+    speakerName: 'Rachel Beveridge',
+    profession: 'Storyteller Panel:',
+    discription: 'Sebastian Maldonado, Kurt Rietema, Sparrow Etter Carlson',
+  },
+  {
+    speakerImage: './images/speaker_03.png',
+    speakerName: 'John Eigege',
+    profession: 'Storyteller Panel:',
+    discription: 'Szabina Sztojka, Andrew Beunk, Laura Osborne',
+  },
+  {
+    speakerImage: './images/speaker_04.png',
+    speakerName: 'Ruth Padilla-DeBorst',
+    profession: 'Storyteller Panel:',
+    discription: 'Mario Matos, John Lee, David Kromminga',
+  },
+];
+
```

Comment:
- [ ] You did great generating this section randomly with javascript :+1: :+1: , But according to the design, this section has a list of six speakers, I'll suggest you update your array to six speakers to match the given design. :+1: :+1: